content/launch-keyboard.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuring your new Launch keyboard
+description:
+  View, add, or modify keys in your OS.
+keywords:
+  - keyboard shortcuts
+  - launch
+  - pop
+  - ubuntu
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+---
+
+## The Launch and Accessories
+
+The following items are included with your Launch:
+
+![Accessories](/files/launch-keyboard/launch-accessories.webp)
+
+## QMK firmware
+
+The Launch keyboard uses our version [QMK firmware](https://github.com/system76/qmk_firmware) and it is built using our changes that are not in upstream [QMK firmware](https://github.com/qmk/qmk_firmware). You can read more about QMK [here](https://github.com/system76/qmk_firmware).
+
+## Layers
+
+The core idea of "Layers" is that one key can be used in different setups for example on our laptops holding the <kbd>Fn</kbd> then pressing the <kbd>F5</kbd> will lower the volume while pressing <kbd>F5</kbd> on it's own will perform a different action like refreshing a web page if you are in a web browser like <u>Firefox</u>. You can read more about layers [here](https://beta.docs.qmk.fm/using-qmk/software-features/feature_layers
```

Comment:
> one key can be used in different setups

"different setups" is vague and probably not the best way to word this.

> The core idea of "Layers" is that one key can be used in different setups for example on our laptops holding the <kbd>Fn</kbd> then pressing the <kbd>F5</kbd> will lower the volume while pressing <kbd>F5</kbd> on it's own will perform a different action like refreshing a web page if you are in a web browser like <u>Firefox</u>.

This is a run-on sentence. I would recommend splitting the sentence before "for example" (and add the appropriate commas in the second sentence.)

> You can read more about layers [here](https://beta.docs.qmk.fm/using-qmk/software-features/feature_layers

The closing parenthesis for this link probably shouldn't be on a separate line.